src/main/java/core/utils/StringAdapter.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.utils;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class StringAdapter {
+
+    /**
+     * Replace characters with accents and 'ñ' to supported characters.
```

Comment:
It should be "Replaces" instead of "Replace"

Reply:
solved it.